api/namex/services/name_request/builders/name_analysis_builder.py

```diff
@@ -635,7 +635,7 @@ def stand_alone_name_different(self, lst_dist_name1, lst_dist_name2, lst_desc_na
                                    stand_alone_words):
         if self.is_standalone_name(lst_desc_name1, stand_alone_words) and self.is_standalone_name(lst_desc_name2,
                                                                                                   stand_alone_words):
-            if lst_dist_name1.__len__() > lst_dist_name2.__len__() or lst_desc_name1.__len__() > lst_desc_name2.__len__():
+            if lst_dist_name1.__len__() != lst_dist_name2.__len__() or lst_desc_name1.__len__() != lst_desc_name2.__len__():
```

Comment:
This feels like it should be in its own function and not embedded in this function since this check.rule an applied to all names.